sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -591,25 +673,25 @@ abstract class ExternalCatalogSuite extends SparkFunSuite with BeforeAndAfterEac
     val catalog = newBasicCatalog()
     val db = catalog.getDatabase("db1")
     val table = CatalogTable(
-      identifier = TableIdentifier("my_table", Some("db1")),
```

Comment:
Could we keep the original names unchanged? Now, the table/database names allow `alphanumeric characters and underscores`. Thus, I think they are named using underscore on purpose?